charts/gateway/README.md

```diff
@@ -1157,6 +1158,51 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
```

Comment:
wording suggestion:

DISKLESS_CONFIG: used to control where configuration values come from. Default is "true". 
When DISKLESS_CONFIG=true, a set of gateway configuration properties are read from the environment variable. 
When DISKLESS_CONFIG=false, the same set of gateway configuration properties will be read from the /opt/SecureSpan/Gateway/node/default/etc/conf/node.properties file